src/renderer/style.scss

```diff
@@ -841,6 +841,8 @@ label.SimpleCheckbox {
 
 .SceneDetail {
   .SceneName {
+    min-width: 10rem;
+    min-height: 2rem;
     display: inline-block;
     margin-top: 0;
```

Comment:
Please set `margin: 0` so that when element has focus, we don't see the outline outside of the top bar.